app/views/layouts/application.html.erb

```diff
@@ -42,10 +42,12 @@
         <div class="grid__item width-five-sixths shift-one-twelfth">
           <div class="toolbar toolbar--wrap-when-small">
               <div class="toolbar__left">
-                <% unless content_for?(:no_back) %>
-                <a class="button button--primary button--small" href="javascript:history.back()">
-                  <i class="icon-arrow_back"></i> <%= t("shared.go_back") %>
-                </a>
+                <% if content_for?(:no_back) %>
```

Comment:
@spokenbird How about the homepage? Looks like `:no_back` is only set on the how page.

Reply:
The home page is rendering the partial hard coded at the moment. I can add a content_for to the home page if that's more desirable?

Reply:
Ah I see. Looks good. :+1: